common/dictionaries.js

```diff
@@ -0,0 +1,9 @@
+export const HDPathsOrAlgosDictionary = {
+    cosmosStandard: [44, 118, 0, 0, 0],
+    terraStation: [44, 330, 0, 0, 0],
+}
+
+export const standardHDPathsOrAlgosDictionary = {
+    cosmos: `cosmosStandard`,
```

Comment:
this is actually ed25519

Reply:
the standard algo for Polkadot is still Schnorrkel. I tried it the other day in polkadot.js and it is still the standard (`sr25519`)